starlette/concurrency.py

```diff
@@ -50,8 +50,9 @@ def _next(iterator: typing.Iterator[T]) -> T:
 
 
 async def iterate_in_threadpool(
-    iterator: typing.Iterator[T],
-) -> typing.AsyncIterator[T]:
+    iterable: typing.Iterable[T],
```

Comment:
This is a breaking change.

Reply:
Okay gotchu. Would you like me to leave the name as `iterator`, and just update the type hint?

Reply:
Yes, please.

Reply:
Alright, renamed back to `iterator`